### PR TITLE
Feature QR Label

### PR DESCRIPTION
**Pull request by CaseyODonnell92**

### CODE REVIEW

File: src/docs/devguide/hugo/site/content/release-notes.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 This page describes changes to the QR Payment Service Interface implemented across different releases of the interface.
 
+## v1.11.0
+Released 18 August 2021
```
Comment (TimElectrum) [inline src/docs/devguide/hugo/site/content/release-notes.md:4]: date reminder

**Review by TimElectrum — APPROVED** [review]
just the date

**Review by mattsegers — CHANGES_REQUESTED** [review]
It looks like we don't have any serialisation/deserialisation tests where we test a "minimally" constructed object (i.e. without optional fields). Can you add some? :) I'm ok with you just adding one for this field

**Comment by CaseyODonnell92** [comment]
> It looks like we don't have any serialisation/deserialisation tests where we test a "minimally" constructed object (i.e. without optional fields). Can you add some? :) I'm ok with you just adding one for this field

I think I'm going to get this out the door now first. But let's please remember to put in some tests out-of-band from this review. Perhaps we should create an issue?

**Pull request merged**